src/Chapter2.hs

```diff
@@ -858,7 +894,9 @@ and reverses it.
   function, but in this task, you need to implement it manually. No
   cheating!
 -}
-rewind = error "rewind: Not Implemented!"
+rewind :: [Int] -> [Int]
```

Comment:
Here you can have a less strict type, as you don't really use the fact that `Int`s are inside in the implementation. So it could be:

```suggestion
rewind :: [a] -> [a]
```